FREELANCERS_AND_PRODUCTION_COMPANIES.md

```diff
@@ -0,0 +1,93 @@
+# Freelancers and productions companies
+
+As a freelancer or production company for Hyper we expect you to follow our guidelines for
+production. Underneath you’ll find and overview of how we work and what we expect from you.
+
+  We hold your opinion in high regard and want to here what you have to bring to the table. But to
+make our work the most effective, especially down the line, we have a set of guidelines.  
+
+## Day to day workflow 
+
+### Design
+
+All our work involve working closely with the whole team at Hyper. But for technology the closest
+ties are to the designers. Their experience with both visual and interaction design paired with
+your knowledge about technology is our edge in developing apps.
+
+They use [Zeplin](https://zeplin.io/) to deliver the designs. This way we don't have to dig trough
+Sketch or Photoshop file to get the correct font-size. Zeplin can be opened in the app on Mac or on
+the web.
+
+### Code
+
+*Regarding code*, we would like to invite you to work in our GitHub. There you’ll get access to the
+repos you need. We will have a internal developer helping you get access and providing access to
```

Comment:
"a internal" or "an internal"? I'm not sure.

Reply:
Pretty sure its "an".

Reply:
👍